assets/js/components/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -1,22 +1,20 @@
 import React, { useEffect, useState } from 'react';
 import { groupBy } from 'lodash';
 
-import { providerData } from '@components/ProviderLabel/ProviderLabel';
+import { PROVIDERS } from '@lib/model';
```

Comment:
Usually uppercase constants are used for primitive values, you may want to rename this :+1:

Reply:
Gonna lowercase it to just`providers` then